docs/tutorials/deploy.md

```diff
@@ -25,10 +25,13 @@ The instructions below are based on the [Firebase quick start][Firebase quick st
     in the root of your project directory. Otherwise, go to
     [Firebase console][Firebase console] to create a new app.
     
-1.  When you initialize your app, you will be prompted for a directory to 
-    use as the public root (default is "public"). Enter `build/bundled` 
-    or `build/unbundled` (supports HTTP/2). `build` contains everything 
-    your application needs to run.
+1.  When you initialize your app, you will be prompted for
+- Database: space
```

Comment:
You need db be enabled

Reply:
Ok, I'll double check Friday and adjust (v1.2 seems to work w/o db enabled).